INIT.md

```diff
@@ -1,10 +1,45 @@
-## webpack init
-
-Through [yeoman](http://yeoman.io/), the `webpack-cli init` feature allows people to create scaffolds and generate new projects quickly. An npm dependency that scaffolds a `webpack.config.js` through `webpack-cli` is what we refer to as an **addon**.
-
-We ask several questions in the default generator to get you started.
-
----
+# webpack-cli init
+
+`webpack-cli init` is used to initialize `webpack` projects quickly by scaffolding configuration and installing modules required for the project as per user preferences.
+
+## Initial Setup
+Follow given steps to setup `webpack-cli init` by installing dependencies:
+1. Create `package.json` through npm
+
+	```shell
+    	$ npm init
```

Comment:
Trim the whitespace in left.

Reply:
Will do it!

Reply:
Never seen `shell` markdown. Is it an alias of `bash`?

Reply:
shell used in github flavoured markdown alias to bash in common markdown